lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -40,7 +47,14 @@ def show_also_part_of_step_nav?
       end
 
       def related_links
-        step_by_step_navs = active_step_by_step? ? [active_step_by_step] : step_navs
+        step_by_step_navs = if active_step_by_step?
+                              [active_step_by_step]
+                            elsif show_related_links_for_secondary_step_by_steps?
+                              secondary_step_by_steps
+                            else
+                              step_navs
```

Comment:
Should this logic be the other way around? So, you check if there are any primary step by steps first, and if there aren't then see about finding any secondary step navs?

Reply:
This is fine because there is a check within the `show_related_links_for_secondary_step_by_steps?` to check that there are no step navs.

If we do decide to change we'd have to do something like this:

```
step_by_step_navs = if active_step_by_step?
                              [active_step_by_step]
                            elsif step_navs.any?
                              step_navs
                            elsif show_related_links_for_secondary_step_by_steps?
                              secondary_step_by_steps
                            else
                              []
                            end
```

which is messy no?

Reply:
I think it makes more sense logic-wise, and makes it more readable to put secondary step navs after, primary ones. 
It also means that you could remove `show_related_links_for_secondary_step_by_steps?` and just use `secondary_step_by_steps.any?`.

Reply:
`show_related_links_for_secondary_step_by_steps?` doesn't check for primary step by steps, it looks like it only checks if there are less than 5 secondary ones.